features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/resources/ResourceHashesEntryDeserializer.kt

```diff
@@ -0,0 +1,32 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.resources
+
+import com.datadog.android.core.internal.persistence.Deserializer
+
+internal class ResourceHashesEntryDeserializer : Deserializer<String, ResourceHashesEntry> {
+    override fun deserialize(model: String): ResourceHashesEntry? {
```

Comment:
> [!CAUTION]
> Such a custom made serialization format seems like it won't allow for any future improvements. I think it'd be better to store data as jsons (we can easily create a local json schema and generate the kotlin implementation like we do for all other events).

Reply:
I've changed to use JSON serialization

Reply:
Why did you make the Json serialization by hand instead of using a Json Schema?

Reply:
I've fixed it now